packages/node_modules/sample-browser-auth-implicit/test/wdio/specs/implicit-grant-authentication.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-undef */
+
+import testUsers from '@ciscospark/test-helper-test-users';
+import {assert} from 'chai';
+
+describe(`sample-browser-auth-implicit`, () => {
+  describe(`authenticate using implicit grant flow`, () => {
+    const browserSpock = browser.select(`browserSpock`);
+    let spock;
+
+    before(`create Spock`, () => testUsers.create({count: 1, config: {displayName: `Spock`}})
+      .then((users) => {
+        [spock] = users;
+      }));
+
+
+    step(`browse to sample app, and verify the user is not authenticated`, () => {
+      browserSpock.url(`/packages/node_modules/sample-browser-auth-implicit`);
+      assert.equal(browserSpock.getText(`#authentication-status`), `Not Authenticated`);
+    });
+
+    step(`initiate implicit grant authentication flow`, () => {
+      browserSpock.click(`[title="authenticate"]`);
+    });
+
+    step(`enter user login information`, () => {
+      // Enter Email
+      browserSpock.waitForExist(`input[type=email]`);
+      browserSpock.setValue(`[placeholder="Email address"]`, spock.email);
+      browserSpock.click(`[name="btnOK"]`);
+
+      // Enter Password
+      browserSpock.waitForExist(`input[type=password]`);
+      browserSpock.setValue(`[placeholder="Password "]`, spock.password);
+      browserSpock.click(`[name="Login.Submit"]`);
+    });
+
+    step(`verify authentication was successfull`, () => {
+      browserSpock.waitUntil(() => browserSpock.getTitle() === `Authentication - Implicit Grant Flow`);
+      assert.equal(browserSpock.getText(`#authentication-status`), `Authenticated`);
+      assert.equal(browserSpock.getAttribute(`[title="logout"]`, `value`), `logout`);
+    });
+
+    step(`logout`, () => {
+      browserSpock.click(`[title="logout"]`);
+      browserSpock.waitUntil(() => browser.select(`browserSpock`).getTitle() === `Authentication - Implicit Grant Flow`);
```

Comment:
To @adamweeks's point earlier, there's no need to reselect spock's browser, right?